lizmap/app/locales/cs_CZ/view/locales/dictionnary.UTF-8.properties

```diff
@@ -192,7 +192,7 @@ digitizing.toolbar.textScale=Velikost
 digitizing.toolbar.import=Importovat v GeoJSON, GPX nebo KML
 digitizing.toolbar.edit=Upravte ( podržte klávesu "Shift" pro výběr více funkcí)
 digitizing.toolbar.erase=Smazat některé funkce
-digitizing.confirme.erase=Opravdu chcete smazat tuto funkci?
```

Comment:
I think this fix must only be made in lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties

@Gustry can you confirm?

Reply:
I think he is right, therefore translations are not lost.
It's not translators fault if we rename the key.